project_report/tex/analysis.tex

```diff
@@ -0,0 +1,208 @@
+\section{Analysis}
+
+\subsection{Use Case analysis}
+
+\subsubsection{Class Candidates}
+
+\subsubsection{Description of Classes}
+
+\subsubsection{UML Analysis Diagram}
+
+\subsection{Use Case Realisation}
+
+\subsubsection{Sequence Diagrams}
+
+            % \begin{enumerate}
+            %     \item A beer type should be selected
+            % \end{enumerate}
+
+\subsubsection{Operation Contracts}
```

Comment:
Please add a description for operation contracts. 
We need to explain why this is relevant for our project, as in, what are we using these for?